src/univariate/continuous/chernoff.jl

```diff
@@ -0,0 +1,320 @@
+end
+
+minimum(d::Chernoff) = -Inf
+maximum(d::Chernoff) = Inf
+insupport(d::Chernoff, x::Real) = isnan(x) ? false : true
+
+mean(d::Chernoff) = 0.0
+var(d::Chernoff) = 0.26355964132470455
+modes(d::Chernoff) = [0.0]
+mode(d::Chernoff) = 0.0
+skewness(d::Chernoff) = 0.0
+kurtosis(d::Chernoff) = -0.16172525511461888
+kurtosis(d::Chernoff, excess::Bool) = kurtosis(d) + (excess ? 0.0 : 3.0) 
+entropy(d::Chernoff) = -0.7515605300273104
+
+### Random number generation
+slowrand(d::Chernoff) = quantile(d, rand(GLOBAL_RNG))
+rand(d::Chernoff) = rand(GLOBAL_RNG, d)      
+function rand(rng::AbstractRNG, d::Chernoff)                 # Ziggurat random number generator --- slow in the tails
+    # constants needed for the Ziggurat algorithm
+    A = 0.03248227216266608
+    x = [
+        1.4765521793744492 
+        1.3583996502410562 
+        1.2788224934376338 
+        1.2167121025431031 
+        1.164660153310361  
+        1.1191528874523227 
+        1.0782281238946987 
+        1.0406685077375248 
+        1.0056599129954287 
+        0.9726255909850547 
+        0.9411372703351518 
+        0.910863725882819  
+        0.8815390956471935 
+        0.8529422519848634 
+        0.8248826278765808 
+        0.7971898990526088 
+        0.769705944382039  
+        0.7422780374708061 
+        0.7147524811697309 
+        0.6869679724643997 
+        0.6587479056872714 
+        0.6298905501492661 
+        0.6001554584431008 
+        0.5692432986584453 
+        0.5367639126935895 
+        0.5021821750911241 
+        0.4647187417226889 
+        0.42314920361072667
+        0.37533885097957154
+        0.31692143952775814
+        0.2358977457249061 
+        1.0218214689661219e-7
+       ]
+    y=[
+        0.02016386420423385
+        0.042162593823411566
+        0.06607475557706186
+        0.09147489698007219
+        0.11817165794330549
+        0.14606157249935905
+        0.17508555351826158
+        0.20521115629454248
+        0.2364240467790298
+        0.26872350680813245
+        0.3021199879039766
+        0.3366338395880132
+        0.37229479658922043
+        0.4091420247190392
+        0.4472246406136998
+        0.48660269400571066
+        0.5273486595545326
+        0.5695495447104147
+        0.6133097942704644
+        0.6587552778727138
+        0.7060388099508409
+        0.7553479187650504
+        0.8069160398536568
+        0.8610391369707446
+        0.9181013320624011
+        0.97861633948841
+        1.0432985878313983
+        1.113195213997703
+        1.1899583790598025
+        1.2764995726449935
+        1.3789927085182485
+        1.516689116183566
+        ]       
+    n = length(x)
+    i = rand(rng, 0:n-1)
+    r = (2.0*rand(rng)-1) * ((i>0) ? x[i] : A/y[1])
+    rabs = abs(r)
+    if rabs < x[i+1]
+        return r 
+    end
+    s = (i>0) ? (y[i]+rand(rng)*(y[i+1]-y[i])) : rand(rng)*y[1]
+    if s < 2.0*f(rabs)
```

Comment:
`f` is defined in ChernoffComputations, so this should be `ChernoffComputations.f`